man/bundle-list.1.txt

```diff
@@ -31,11 +31,12 @@ OPTIONS
        --paths
 	      Print the path to each gem in the bundle.
 
-       --without-group
-	      Print all gems expect from a group.
+       --without-group=<list>
+	      A space-separated list of groups referencing gems to skip during
```

Comment:
Super minor change but I think "groups of gems to skip during printing" reads better than "groups referencing gems to skip during printing".